activesupport/CHANGELOG.md

```diff
@@ -28,4 +28,8 @@
 
     *Taketo Takashima*
 
+*   Make `ActiveSupport::FileUpdateChecker` faster when checking many file-extensions.
+
+    *Jonathan del Strother*
+
```

Comment:
@jdelStrother Changes need to go at the top of the file, for next time :pray:

Reply:
Oops, sorry